lib/salvator.js

```diff
@@ -84,8 +85,10 @@ module.exports = function(filepath, options) {
                     return next(new Error("Limit exceeded"));
                 }
 
-                fs.exists(filepath, function(exists) {
-                    next(null, exists);
+                fs.stat(filepath, function (err) {
```

Comment:
Sure, this is clearly better! I'm gonna correct it and submit an updated PR.

Thanks!